riff-raff.yaml

```diff
@@ -4,4 +4,14 @@ deployments:
   prism:
     type: autoscaling
     parameters:
-      bucket: deploy-tools-dist
\ No newline at end of file
+      bucket: deploy-tools-dist
+    dependencies:
+      - update-ami
+  update-ami:
+    type: ami-cloudformation-parameter
+    app: prism
+    parameters:
+      amiParameter: MachineImageAMI
+      amiTags:
+        Recipe: xenial-java8-deploy-infrastructure
+        AmigoStage: PROD
```

Comment:
Sorry for not replying to this. For the record we discussed this and my argument was that `AmigoStage` is a proxy for this, but it's not the main point of the tag. In hindsight this should be `Stage` and then we should use `BuiltBy` to determine what platform built it.

I guess the other side of the argument would be that we don't care what builds it, we just want the latest version.